v-next/core/src/internal/tasks/resolved-task.ts

```diff
@@ -237,18 +241,31 @@ export class ResolvedTask implements Task {
    */
   async #resolveFileAction(
     actionFileUrl: string,
-    taskId: string[],
   ): Promise<NewTaskActionFunction | TaskOverrideActionFunction> {
     let resolvedActionFn;
     try {
       resolvedActionFn = await import(actionFileUrl);
     } catch (error) {
       ensureError(error);
+
+      if (this.pluginId !== undefined) {
```

Comment:
I believe this logic is incorrect. Because it always validates the first plugin that defined this task. We should validate the one defining the current action instead.